src/styles.css

```diff
@@ -0,0 +1,447 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+*,
+html,
+/* body {
```

Comment:
[OPTIONAL] Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional.